### DISCUSSION TITLE
[Script Editor] Pressing space or period should apply the highlighted autocomplete suggestion

### DISCUSSION THREAD

**Discussion by Kartopod** [godotengine/godot-proposals#8097] [Editor]
At the moment, when you hit space or period, the symbol is simply entered. It would be neat if pressing either of those would complete the suggestion and then enter the symbol. I believe this is the case with other code editors, would be a nice feature to have in the godot script editor too! Maybe even add a toggle in the editor settings to disable it based on preference.

**Comment by Calinou** [comment]
You can change the keyboard shortcuts used to accept autocompletion in the Editor Settings' Shortcuts tab:

![image](https://github.com/godotengine/godot-proposals/assets/180032/b172b404-39ac-4da3-a076-e4e29745027e)
